### ISSUE TITLE
Web hook has URL instead of API key

### ISSUE THREAD

**Issue by luntzel** [risk-io/chef-slack#4]
Unless I'm missing something, this cannot work with the current Slack setup for web hooks. Please advise. Thanks!

**Comment by aaronsky** [comment]
Is this still an issue?

**Comment by luntzel** [comment]
The last commit was Sep 16, 2014. I think it is still an issue.

**Comment by luntzel** [comment]
Incoming web hooks just have URL's now instead of an API key, like so:

https://hooks.slack.com/services/T054FL27M/B0473LEC5/6sMnY01X....etc

If nobody else is using this cookbook anyway...I'll spend some time to try to make it work.

**Comment by jro** [comment]
We'd love a PR. We do use the cookbook and slackr internally but it seems like slack must have setup redirection for existing endpoints as things haven't broken for us.

**Comment by luntzel** [comment]
Yeah it seems to be the case that older API endpoints are preserved. Putting together something new is another beast. I'll give it a whirl.

**Comment by ngtjah** [comment]
Yes, not working for me.

Slack.com - 404 - No hooks

**Comment by spuder** [comment]
+1 

New integrations give a 404, 

I was able to work around it by stripping off the first part of the incoming web hook url (Use everything to the right of the final `/` )


![screenshot 2015-12-02 10 32 09](https://cloud.githubusercontent.com/assets/242382/11538652/242f04a2-98e0-11e5-8d45-67ddc5818d96.png)
